PocketKit/Sources/PocketKit/MyList/SavedItemsList/SavedItemsListViewModel.swift

```diff
@@ -435,7 +435,14 @@ extension SavedItemsListViewModel {
     }
 
     private func applySorting() {
-        let sortDescriptorTemp = NSSortDescriptor(keyPath: \SavedItem.createdAt, ascending: (listOptions.selectedSort == .oldest))
+        var sortDescriptorTemp: NSSortDescriptor!
```

Comment:
should we make this optional instead of forced unwrapped? do we want to think about adding `force_unwrapping` to our SwiftLint rule? @Pocket/ios

Reply:
+1 for using an optional here. Force unwraps are always a crash risk

Reply:
thanks @cyndichin , this is fixed.